docs/installation/basic_installation.md

```diff
@@ -20,19 +20,19 @@ If you would prefer less automation, or you  already have existing services on y
 
     <p>Using docker, all of SeAT's dependencies is neatly abstracted away from your hosts operating system and does not conflict with any pre-existing software you might have on your server. With a little bit of extra work, Docker installations are also possible on Windows.
     <br><br>
-    If you have either <code>docker</code> or <code>docker-compose</code> already installed on your server, make sure you have the latest versions as detailed on the <a href="/installation/requirements/">requirements</a> page. If you do not have them installed yet, the following script will install them if needed and then download the SeAT <code>docker-compose.yml</code> and <code>.env</code> files to <code>/opt/seat-docker</code> on your server:</p>
+    If you have either <code>docker</code> or <code>docker-compose</code> already installed on your server, make sure you have the latest versions as detailed on the [requirements](/installation/requirements) page. If you do not have them installed yet, the following script will install them if needed and then download the SeAT <code>docker-compose.yml</code> and <code>.env</code> files to <code>/opt/seat-docker</code> on your server:</p>
```

Comment:
These section is part of the `<mdc-tabs>` therefore the markdown link-creation does not compile and need to be raw html. 
You need to change this back